R/proposicoes.R

```diff
@@ -477,7 +477,7 @@ fetch_ids_relacionadas <- function(id, casa) {
 #' @param id_prop proposition's ID
 #' @param filter_texto_materia Bool flag to filter the initial texts
 #' @param casa senado ou congresso
-#' @return dataframe 
+#' @return dataframe
 #' @rdname scrap_senado_congresso_documentos
 #' @export
 scrap_senado_congresso_documentos <- function(id_prop, filter_texto_materia = F, casa) {
```

Comment:
Como o parâmetro filter_texto_materia tem um valor default, deve vir por último na lista, pra que se a pessoa colocar a casa sem especificar o nome do parâmetro, ele identifique. Quando fui rodar assim falhou:
rcongresso::scrap_senado_congresso_documentos(91341,'senado')
Error in tolower(casa) : argument "casa" is missing, with no default